src/main/kotlin/racing/domain/Car.kt

```diff
@@ -0,0 +1,18 @@
+package racing.domain
+
+data class Car(
+    val name: CarName,
+    private var _route: Int = 0
+) {
+    public val route: Int
+        get() {
+            if (_route == 0) {
+                _route = 0
+            }
+            return _route
+        }
```

Comment:
다음과 같이 작성할 수 있을 것 같아요!
```suggestion
    val route: Int
        get() = _route
```